android/app-newm/src/main/java/io/newm/NewmAppComposable.kt

```diff
@@ -89,6 +89,7 @@ val initialScreen = Screen.NFTLibrary
 internal fun NewmApp(
     logger: NewmAppLogger,
     eventLogger: NewmAppEventLogger,
+    showRecordStore: Boolean = false //update once backend supports special access
```

Comment:
Setting it by default to false until we find a good way to gate features for early preview.